pkgs/by-name/an/anytype/anytype-heart/default.nix

```diff
@@ -12,12 +12,12 @@
 let
   tantivy-go = callPackage ../tantivy-go { };
   pname = "anytype-heart";
-  version = "0.39.10";
+  version = "0.40.0-spacestore";
```

Comment:
This was released last week, two weeks after `0.45.3`. I think we should use `0.39.11` to be safe, since this is a patch version update.